frontend/javascripts/router.tsx

```diff
@@ -630,7 +630,17 @@ class ReactRouter extends React.Component<Props> {
               <SecuredRouteWithErrorBoundary
                 isAuthenticated={isAuthenticated}
                 path="/organizations/:organizationId"
-                render={() => <OrganizationEditView />}
+                render={() => <Redirect to="/organization" />}
```

Comment:
I left the legacy frontend route for backward compatibility. Maybe that is not really needed though.